scripts/sql/222_image_exporter_gcp_support.up.sql

```diff
@@ -0,0 +1,76 @@
+UPDATE plugin_pipeline_script SET script=E'#!/bin/bash
+    set -eo pipefail
+echo $GoogleServiceAccount > output.txt
+cat output.txt| base64 -d > gcloud.json    
+
+export platform=$(echo $CI_CD_EVENT | jq --raw-output .commonWorkflowRequest.ciBuildConfig.dockerBuildConfig.targetPlatform)
+echo $platform
+arch
+if [[ $platform == "linux/arm64,linux/amd64" ]] ; then
+    platform=$Platform
+elif [[ $platform == "linux/arm64" ]]
+then
+    platform="arm64"
+elif [[ $platform == "linux/amd64" ]]  
+then
+    platform="amd64"
+else
+    if [[ arch == "x86_64" ]]
+    then
+        platform="amd64"
+    else
+        platform="arm64"
+    fi    
+fi        
+echo $platform   
+CloudProvider=$(echo "$CloudProvider" | awk \'{print tolower($0)}\')
+current_timestamp=$(date +%s)
+if [[ -z $FilePrefix ]]
+then
+    file=$ContainerImage.tar
+    file=$(echo $file | tr \'/\' \'_\')
+else
+    file=$FilePrefix-$ContainerImage.tar
+    file=$(echo $file | tr \'/\' \'_\')
+fi   
+echo $file     
+future_timestamp=$((current_timestamp + $Expiry * 60))
+future_date=$(date -u  -d@"$future_timestamp" +"%Y-%m-%dT%H:%M:%SZ")
+aws_secs=$(($Expiry * 60))
+gcp_secs="${Expiry}m"
+docker pull --platform linux/$platform $ContainerImage
+docker save $ContainerImage > $file
+ls
+if [ $CloudProvider == "azure" ]
+then
+    docker run  --rm  -v $(pwd):/data  mcr.microsoft.com/azure-cli  /bin/bash -c " az storage blob upload --account-name $AzureAccountName --account-key $AzureAccountKey  --container-name $BucketName --name $file --file data/$file"
+    echo "docker run  --rm   mcr.microsoft.com/azure-cli  /bin/bash -c " az storage blob generate-sas --account-name $AzureAccountName --account-key $AzureAccountKey --container-name $BucketName --name $file --permissions r --expiry $future_date""
+    sas_token=$(docker run  --rm   mcr.microsoft.com/azure-cli  /bin/bash -c " az storage blob generate-sas --account-name $AzureAccountName --account-key $AzureAccountKey --container-name $BucketName --name $file --permissions r --expiry $future_date")
+    token=$sas_token
+    echo $token 
+    token=$(echo $sas_token| tr -d \'"\')
+    echo $token
+    link=https://$AzureAccountName.blob.core.windows.net/$BucketName/$file?$token
+fi
+if [ $CloudProvider == "aws" ]
+then
+    echo "aws command"
+    docker run --network=host --rm -v $(pwd):/data -e AWS_ACCESS_KEY_ID=$AwsAccessKey -e AWS_SECRET_ACCESS_KEY=$AwsSecretKey amazon/aws-cli:latest  s3 cp /data/$file s3://$BucketName --region $AwsRegion
```

Comment:
Use this image instead of image from docker hub - public.ecr.aws/aws-cli/aws-cli:latest